IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -220,22 +220,22 @@ - (BOOL)removeItemsWithKey:(MSIDTokenCacheKey *)key
     {
         [query setObject:key.account forKey:(id)kSecAttrAccount];
     }
-    if (key.type)
+    if (key.type != nil)
```

Comment:
just wondering: did xcode put warning here?

Reply:
It's an addition to clang static analyzer. See details here: https://clang.llvm.org/doxygen/NumberObjectConversionChecker_8cpp_source.html